src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -570,6 +630,19 @@ public override ValueTask WriteAsync(ReadOnlyMemory<byte> buffer, CancellationTo
                         return new ValueTask(Task.FromException(new ObjectDisposedException(nameof(Http2WriteStream))));
                     }
 
+                    if (_abortStream)
+                    {
+                        return new ValueTask();
+                    }
+
+                    if (http2Stream._abortRequestBody)
+                    {
+                        // If are asked to abort sending request body after we started, send RST and ignore test for the stream.
```

Comment:
nit:  should be something like "If asked to abort sending request body after we started, send RST and ignore test for the stream."

And should this "ignore test for the stream" be changed to "ignore the rest of the stream"?  I don't know enough about what this code is doing.